SPECS/node-problem-detector/001-remove_arm64_build.patch

```diff
@@ -0,0 +1,56 @@
+diff -Naur a/Makefile b/Makefile
+--- a/Makefile	2021-05-14 17:05:59.000000000 -0700
++++ b/Makefile	2022-02-24 11:53:35.220842944 -0800
+@@ -22,8 +22,8 @@
+ all: build
+ 
+ # PLATFORMS is the set of OS_ARCH that NPD can build against.
+-LINUX_PLATFORMS=linux_amd64 linux_arm64
```

Comment:
Now I'm also curious: why do we want to not build for ARM64? We do support that platform.

Reply:
Arm64 was refusing to build after upgrading. I spoke with Henry Beberman who mentioned that the end use case for this package does not need ARM64 and it's a supplementary package not part of the core featureset.